easybuild/easyconfigs/g/GATE/GATE-6.2-intel-2015a.eb

```diff
@@ -0,0 +1,23 @@
+name = 'GATE'
+version = '6.2'
+
+homepage = 'http://www.opengatecollaboration.org/'
+description = """GATE is an advanced opensource software developed by the international OpenGATE collaboration and
+ dedicated to the numerical simulations in medical imaging. It currently supports simulations of Emission Tomography
+  (Positron Emission Tomography - PET and Single Photon Emission Computed Tomography - SPECT), and Computed Tomography"""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+sources = [('%%(namelower)s_v%s_tar_gz_15843.gz' % '_'.join(version.split('.')), "tar xfvz %s")]
+source_urls = ['http://www.opengatecollaboration.org/sites/opengatecollaboration.org/files/gate_release/2012/08/']
+
+patches = ['GATE-%(version)s_Makefile-prefix.patch']
+
+dependencies = [
+    ('Geant4', '9.5.p01'),
+    ('CLHEP', '2.1.1.0'),
+    ('ROOT', 'v5.34.01'),
+    ('CMake', '2.8.4'),
```

Comment:
should be build dep indeed, will fix

old version to keep this in sync with existing GATE 6.2 easyconfigs (this is just a different toolchain)